PFOffer.sol

```diff
@@ -182,10 +182,12 @@ contract PFOffer {
     }
 
     function sign() {
+        var (recipient,,,votingDeadline,,) = client.proposals(proposalID);
```

Comment:
`recipient` is not used so you can replace it with a `_`. Don't leave it empty, the contract won't compile then.